ledger/sandbox-common/src/test/lib/scala/platform/sandbox/AbstractSandboxFixture.scala

```diff
@@ -63,16 +68,22 @@ trait AbstractSandboxFixture extends AkkaBeforeAndAfterAll {
       .fold[TimeProvider](_ => TimeProvider.UTC, Await.result(_, 30.seconds))
   }
 
-  protected def config: SandboxConfig =
-    SandboxConfig.defaultConfig.copy(
-      port = Port.Dynamic,
```

Comment:
I may be mistaken but I think by not specifying the port here, we will get the `ApiServerConfig.DefaultPort` which maps to 6865. It may not be a problem since the tests pass, but I can imagine that unit the tests might start clashing in CI because of that.

Reply:
Same comment applies to several other such places in this PR

Reply:
`SandboxOnXForTest` defines port as 0 for all tests

Reply:
Right, haven't noticed that